projects/Glados-PortalNetwork.md

```diff
@@ -0,0 +1,76 @@
+
+## Specification
+
+- Graphs: Create a list of interesting data that would be useful and helpful to glados if it was displayed on the page, then figure how that information would be best suited to be display on glados by making different graphs best suited for the information i'm displaying, then figure out how to pull and display the information I want to display through the graphs using sea-orm or sea-query. If these graphs are useful and insightful toward glados and the team approves of them they might be implemented
+
+
+- Tests: When we encounter a bug or problem within glados it sometimes may stop glados from working or stop us from being able to view the health of the portal network which can sometimes be a problem in its own, so it is sometimes a good idea to write tests to ensure to do not encounter these same bugs again which can be very important.
+
+
+- UI: Create a ruff daft using figma of how I would improve the ui run it over with the Portal Network team and if they approve of it begin styling it in my ide.
+
+
+- Dynamic loading: Instead of everything loading all at once, it will load one thing at a time to improve performance and time complexity.
+
+
+- (If I have extra time) Lower Level Features: This is to be determined on how my progress goes during the fellowship. But would be a bonus depending how many project goes and would be a growth opportunity for me.
+
+## Road Map
+
+- 1st Phase: Research a ton into the ethereum, the portal network, glados and into my project. Continue to contribute to glados
+
+
+- 2nd: Phase: Have a really good idea into what I’m to do for my project and have completed my project proposal, as well as continue to contribute and improve glados in its entirety.
+
+
+- 3rd: Phase: By this point I should have made 3 to 4 significant contributions which has improved glados overall
+
+
+- 4th: Phase: By this point I should have made 5 to 7 significant contributions in which has improved glados overall
+
+
+- 5th: Phase: By this point I should have made 10 to 12 significant contributions which improved glados overall and created many way’s for the users to be able to easily view and visualize the health the portal network through the contributions I have done on glados. 
+
+
+- 6th: (Optional Phase if I have extra time): deep dive into the lower level workings of glados to add new features or optimizations
+
+## Possible Challenges
+
+- Some of the possible challenges I believe that I may occur are facing some learning curves along the way which may prevent me from getting PR’s out as fast and temporary halt my learning and progress along the way.
```

Comment:
Do you see any specific learning curves? What are the things that are currently holding you back?